pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go

```diff
@@ -93,9 +89,37 @@ func (w *Watcher) Start(stopCh <-chan struct{}) error {
 		}
 	}(fsWatcher)
 
+	go wait.Until(w.handlerPluginEvent, time.Second, stopCh)
 	return nil
 }
 
+func (w *Watcher) handlerPluginEvent() {
+	for w.processPluginEvent() {
+	}
+}
```

Comment:
Why do you need an extra function to implement a loop? Can't it be done in the `processPluginEvent`?